test/Test/TypeRep/MapProperty.hs

```diff
@@ -83,7 +83,7 @@ test_DeleteInvariant = prop "invariantCheck (delete k b) == True" $ do
 -- it doesn't compare values so it's not true equality. But this should be
 -- enough for tests.
 newtype FpMap f = FpMap (TypeRepMap f)
-  deriving (Show, Semigroup, Monoid)
+  deriving stock (Show, Semigroup, Monoid)
```

Comment:
I think `newtyp` strategy is more appropriate here
```suggestion
  deriving newtype (Show, Semigroup, Monoid)
```

Reply:
Yes, agreed